src/formatter.rs

```diff
@@ -151,7 +180,8 @@ where
 }
 
 /// A formatter that formats span fields into logfmt.
-pub struct FieldsFormatter;
+#[derive(Default)]
```

Comment:
Otherwise users can construct this type directly which means we cannot add fields to it without breaking changes.

```suggestion
#[derive(Default)]
#[non_exhaustive]
```